ncm-ceph/src/main/perl/Ceph/commands.pm

```diff
@@ -161,8 +161,8 @@ sub ssh_known_keys {
         if (!$lines) {
```

Comment:
Sorry for looking at something that is not in the diff, but whaty's the purpose of this?  It looks like you only care on whether the output is empty?